modules/core/src/main/scala/com/caesars/tracing/TracingUtils.scala

```diff
@@ -0,0 +1,55 @@
+package com.caesars.tracing
+
+import io.janstenpickle.trace4cats.inject.EntryPoint
+import io.janstenpickle.trace4cats.kernel.{SpanCompleter, SpanSampler}
+import io.janstenpickle.trace4cats.model.TraceProcess
+import io.janstenpickle.trace4cats.newrelic.{Endpoint, NewRelicSpanCompleter}
+import org.asynchttpclient.DefaultAsyncHttpClientConfig
+import org.http4s.asynchttpclient.client.AsyncHttpClient
+import org.http4s.client.Client
+import org.http4s.client.middleware.Logger
+import zio.*
+import zio.interop.catz.*
+import zio.interop.catz.implicits.rts
+
+object TracingUtils {
+  private val sampler: ULayer[Has[SpanSampler[Task]]] = ZLayer.succeed(SpanSampler.always[Task])
```

Comment:
My only experience with tracing is reading half of https://learning.oreilly.com/library/view/distributed-tracing-in/9781492056621/. I recall that book cautioning against always sampling since it can produce an unmanageable load. For our purposes today, I think always is OK, but, in general, it seems reasonable to me for it to be configurable.

What are your thoughts, Alex?

Reply:
I concur re: making it configurable, but would like to not make any changes to the existing code here.  I'll raise this as an issue on the project.

Reply:
I agree with this as well, however, there is one use case for using `always` - New Relic has this feature called infinite-tracing where you send everything to their infinite-tracing server (different from the normal New Relic API) and that server makes the decision - this is New Relic's implementation of [tail sampling](https://grafana.com/docs/tempo/latest/grafana-agent/tail-based-sampling/) where their server makes a decision about whether to show the entire trace in New Relic - this is usually the only case to set sampling to always

Reply:
If I'm reading your comment correctly, Calvin, you're stating that `Always` is appropriate for NR's infinite-tracing. However, you seem to agree making it configurable, i.e. an input, so that the user can decide? If yes, should we create a ticket on our side or perhaps an issue in this repo to implement it?

Reply:
Istio also applies its [own filtering](https://preliminary.istio.io/latest/about/faq/distributed-tracing/) btw. By default, it only samples 1% of spans. That is the case on USD